README.md

```diff
@@ -16,6 +16,15 @@ We use [rust-clippy](https://github.com/rust-lang-nursery/rust-clippy) linters t
 
 There are plenty of [IDEs](https://areweideyet.com) and other [Rust development tools to consider](https://github.com/rust-unofficial/awesome-rust#development-tools).
 
+### Description
+
+This crate is an sha3 implementation of the Hashing trait, defined in the libHash repository:
+(https://github.com/Fantom-foundation/libhash). This hash will be used for the consensus algorithms,
+such as the Directed Acyclic Graph (DAG) algorithm (https://github.com/Fantom-foundation/libconsensus-dag).
+
+This crate provides access to a single struct: Hash. Hash stores a reference vector of 32 8 bit 
+unsigned integers. These values are then converted to a hash using the sha3 hashing algorithm. 
```

Comment:
The correct spelling is "SHA-3", with uppercase letters because it's an acronym.